.github/workflows/schema.yml

```diff
@@ -0,0 +1,32 @@
+name: Update JSON schema 
+
+on:
+  push:
+		paths:
+		- ../../schemas/'schema.json'
+
+jobs:
+  update-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/checkout@v2
+        with:
+          repository: jina-ai/api
+          path: schema
+          token: ${{ secrets.JINA_DEV_BOT }}
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: copy-schema-to-api-repo
+        run: |
+          python3 ../../scripts/pydanticJSONgenerator.py
+          mv ../../schema.json ../../api/schemas/schema.json
```

Comment:
- `../../api/schemas/schema.json` Please note that the local path for cloning is `schema` not `api`.
- Can you rename the final file to something like `hub_executor_schema.json`, `schema.json` is too generic?

Reply:
Done